crates/turborepo-telemetry/src/client.rs

```diff
@@ -0,0 +1,71 @@
+use std::env;
```

Comment:
Should this go in the api-client crate? It feels a little confusing to have code like `create_request_builder` in the telemetry crate?

Reply:
👍🏼 yea good call - moved!